docsite/rst/playbooks_filters.rst

```diff
@@ -7,6 +7,8 @@ Jinja2 filters
 Filters in Jinja2 are a way of transforming template expressions from one kind of data into another.  Jinja2
 ships with many of these. See `builtin filters`_ in the official Jinja2 template documentation.
 
+Take into account that filters always execute on the ansible controller, NOT on the task target, they manipulate local data.
```

Comment:
Suggest: Take into account that filters always execute on the Ansible controller, **not** on the task target, as they manipulate local data.